.gitignore

```diff
@@ -4,3 +4,4 @@ vendor
 tests/generated
 TAG_MESSAGE
 composer.lock
+.idea/
```

Comment:
This change is not relevant for the pull request. Please remove it.

Developers should keep ignores relevant to their local development environment in their own [global `.gitignore` file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).